### PR TITLE
Enhance requirement

### PR DESCRIPTION
**Pull request by ndthanhdev**

### CODE REVIEW

**Comment by vunguyenhung** [comment]
Too many moving parts, please consider making a monolith application (called `OBN-Tracker`) first for an easier implementation. Breaking them to micro-services is the "optimization" step & it takes time (believe me, I've gone through that path)

**Comment by ndthanhdev** [comment]
I think you & me can though that path again.

**Comment by vunguyenhung** [comment]
@ndthanhdev “Premature optimization is the root of all evil”

**Comment by ndthanhdev** [comment]
Ok. Let place it here. So we won't forget it.

File: requirements/operator.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Operator
```
Comment (ndthanhdev) [inline requirements/operator.md:1]: Add later

**Pull request closed without merging**